.github/workflows/docker-image.yml

```diff
@@ -26,28 +26,24 @@ jobs:
         uses: actions/checkout@v3
       # Uses the `docker/login-action` action to log in to the Container registry registry using the account and password that will publish the packages. Once published, the packages are scoped to the account defined here.
       - name: Log in to the Container registry
-        uses: docker/login-action@65b78e6e13532edd9afa3aa52ac7964289d1a9c1
+        uses: docker/login-action@v2
```

Comment:
do we already have a v1

Reply:
Not ours, it's from https://github.com/docker/login-action